governance.md

```diff
@@ -0,0 +1,393 @@
+interests include, but are not limited to:
+
+-   Financial interests, such as investments, employment, or contracting work,
+    outside of The Project that may influence their work on The Project.
+-   Access to proprietary information of their employer that could potentially
+    leak into their work with the Project.
+-   An issue where the person privately gains an advantage from The Project
+    resources, but The Project has no gain or suffers a disadvantage.
+
+All members of the Council, BDFL included, shall disclose to the rest of the
+Council any conflict of interest they may have. Members with a conflict of
+interest in a particular issue may participate in Council discussions on that
+issue, but must recuse themselves from voting on the issue. If the BDFL has
+recused his/herself for a particular decision, they will appoint a substitute
+BDFL for that decision.
+
+
+### Voting
+
+In general, the Council makes decisions by lazy consensus with a minimum of
+participation based on the importance of the decisions to be made.
+Conversations happen on public GitHub issues for most cases, and
+through private e-mail for more sensitive issues.
+
+To make a decision the Council discusses the topic, a proposal is made, and a
+suitable wait time occurs for Council members to either agree or veto.
+By consensus we mean that any Council member can veto a decision with
+justification.  By lazy we mean that not all Council members must participate,
+and that absence is interpreted as assent.  By a minimum of participation we
+mean that we require the participation of a certain number of individuals based
+on the importance of the issue; for non-contentious issues a single Council
+member may move forward after a suitable time, while for contentious issues we
+will often require a few, often from different institutions.
+
+The interested reader may wish to also read
+[NumPy governance section](https://docs.scipy.org/doc/numpy/dev/governance/governance.html#consensus-based-decision-making-by-the-community)
+on this topic.
+
+
+### Private communications of the Council
+
+Unless specifically required, all Council discussions and activities will be
+public and done in collaboration and discussion with the Project Contributors
+and Community. The Council will have a private mailing list that will be used
+sparingly and only when a specific matter requires privacy. When private
+communications and decisions are needed, the Council will do its best to
+summarize those to the Community after eliding personal/private/sensitive
+information that should not be posted to the public internet.
+
+### Subcommittees
+
+The Council can create subcommittees that provide leadership and guidance for
+specific aspects of the project. Subcommittees are created using the standard
+voting procedure (e.g. lazy consensus). The members of the subcommittee need
+not all be Steering Council Members, but they must have at least one Steering
```

Comment:
What's the rationale for requiring a steering council member to be in the subcommittee?

Reply:
Let's keep the discussion in the thread above to avoid discussing it twice. We can update this text based on whatever consensus we reach.